bciers/apps/registration/app/page.tsx

```diff
@@ -120,6 +121,14 @@ export default function Page() {
         },
       }}
     >
+      <Note variant="important">
+        <b>Important:</b> It can be used to highlight important information or
```

Comment:
Do the important notes always have "Important" in front of them, and regular notes "Note"? If so, we could add this text to the component itself based on the variant

Reply:
I thought about that and am not opposed to it, but I thought a general component that would allow the user to add whatever they need to in it is more flexible as there are some `Info` variants in the design that have multiple bolded text areas. It's not difficult to add a `<b>Important:</b>` and maybe there would be a future case where they would want an icon to start or something else. 

The argument against that is consistency, and we could easily allow them to pass a React node for further bolded text or whatever else is needed after the `Important:` prefix. 

I'll leave this at your mercy haha.

Reply:
@BCerki